components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -263,6 +263,8 @@ tweaking their $case/namelist_scream.xml file.
     <Filename hgrid="ne256np4">${DIN_LOC_ROOT}/atm/scream/init/screami_ne256np4L128_ifs-20200120_20220914.nc</Filename>
     <Filename hgrid="ne512np4">${DIN_LOC_ROOT}/atm/scream/init/screami_ne512np4L128_20220823.nc</Filename> 
     <Filename hgrid="ne1024np4">${DIN_LOC_ROOT}/atm/scream/init/screami_ne1024np4L128_era5-20131001-topoadj-16x_20220914.nc</Filename>
+    <Filename hgrid="ne1024np4" COMPSET=".*SCREAM%DYAMOND-1_ELM">${DIN_LOC_ROOT}/atm/scream/init/screami_ne1024np4L128_ifs-20160801-topoadj16x_20221011.nc</Filename>
+    <Filename hgrid="ne1024np4" COMPSET=".*SCREAM%DYAMOND-1-X6T_ELM">${DIN_LOC_ROOT}/atm/scream/init/screami_ne1024np4L128_ifs-20160801-topoadjx6t_20221011.nc</Filename>
```

Comment:
Sanity check question - topography is still included in the screami file, right? If so, then this X6T line is perfect.

Reply:
Yes, Peter, topo included in in the screami. The file to be used will be automatically set via compset, F2010-SCREAMv1-DYAMOND1 or F2010-SCREAMv1-DYAMOND1-X6T.